app/main.py

```diff
@@ -1 +1,35 @@
-# write your code here
+from __future__ import annotations
+
+
+class Animal:
+    alive = []
+
+    def __init__(self,
+                 name: str,
+                 health: int = 100,
+                 hidden: bool = False) -> None:
+        self.name = name
+        self.health = health
+        self.hidden = hidden
+        Animal.alive.append(self)
+
+    def __repr__(self) -> str:
+        tp = self
+        return f"{{Name: {tp.name}, Health: {tp.health}, Hidden: {tp.hidden}}}"
```

Comment:
not sure if tp is useful here. maybe to can skip this step and use self.name

Reply:
I change self.name to t(em)p.name for no tearing of the tape and have length less than the maximum of flake8

Reply:
Yes, it's a redundant variable. Just use brackets to wrap f-string according to flake8